src/CoreShop/Bundle/ResourceBundle/Doctrine/ORM/EntityMerger.php

```diff
@@ -278,7 +294,14 @@ private function cascadeMerge($entity, array &$visited): void
     {
         $class = $this->em->getClassMetadata($entity::class);
 
-        foreach ($class->associationMappings as $assoc) {
+        $associationMappings = array_filter(
```

Comment:
@solverat this and the mapping files was the reason for the re-added unit-definition